### ISSUE TITLE
does mods work?

### ISSUE THREAD

**Issue by DrCoomerPhD** [sultim-t/xash-rt#108]
stupid question but i bet not

**Comment by slambo311** [comment]
No, from what I understand you'd have to make modifications to the map itself and whatnot in order to make them work.
